ev3/legoev3_motor.c

```diff
@@ -955,6 +963,7 @@ static int legoev3_motor_probe(struct lego_device *ldev)
 	hrtimer_init(&ev3_tm->timer, CLOCK_MONOTONIC, HRTIMER_MODE_REL);
 	ev3_tm->timer.function = legoev3_motor_timer_callback;
 
+	ev3_tm->tm.oldstate = 0;
```

Comment:
This is not necessary. The `struct tacho_motor_device` is zeroed when it is allocated, so this will already be 0.